akka-http-core/src/main/scala/akka/http/scaladsl/model/Multipart.scala

```diff
@@ -43,20 +53,48 @@ sealed trait Multipart {
         .flatMapConcat(ConstantFun.scalaIdentityFunction)
     HttpEntity.Chunked(mediaType withBoundary boundary withCharset charset, chunks)
   }
+
+  /** Java API */
```

Comment:
Please never use this comment: it is the most useless thing that can show up in the method summary of JavaDoc, and it is also asymmetric with the “Scala API”.